community-containers/dlna/readme.md

```diff
@@ -0,0 +1,9 @@
+## DLNA server
+This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server Nextcloud-AIO in your local network.
+
```

Comment:
You will need to add some hints here, some ideas what should be written here are in https://github.com/nextcloud/all-in-one/tree/main/community-containers/plex#notes

Reply:
OK, how about now? I've also added a note about the existing nextcloud container requirement.